manager_test.go

```diff
@@ -152,3 +155,54 @@ func TestManager_getTracefsRegex(t *testing.T) {
 		})
 	}
 }
+
+func TestDumpMaps(t *testing.T) {
+	err := rlimit.RemoveMemlock()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	m := &Manager{
+		Probes: []*Probe{
+			{ProbeIdentificationPair: ProbeIdentificationPair{EBPFFuncName: "access_map_one"}},
+		},
+		Maps: []*Map{
+			{Name: "map_one"},
+		},
+	}
+
+	opts := Options{
+		RLimit: &unix.Rlimit{
+			Cur: math.MaxUint64,
+			Max: math.MaxUint64,
+		},
+		ExcludedFunctions: []string{"access_map_two"},
+	}
+
+	f, err := os.Open("testdata/exclude.elf")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { _ = f.Close() })
+
+	err = m.InitWithOptions(f, opts)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	dumpContents := "mapdump"
+
+	m.DumpHandler = func(_ *Manager, mapName string, currentMap *ebpf.Map, w io.Writer) {
+		_, _ = io.WriteString(w, dumpContents)
+	}
+
+	var output bytes.Buffer
+	err = m.DumpMaps(&output, "map_one")
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.Equal(t, dumpContents, output.String())
```

Comment:
Since you are doing just the single `assert`, can you do this instead? That way we don't have to add any new dependencies to this library.
```suggestion
	if dumpContents != output.String() {
		t.Error("expected %s, got %s", dumpContents, output.String())
	}
```